[ReutersBridge] Change timestamp, add new feed, add alt text to image.

csisoap opened the pull request:
- Change timestamp for each article from modified date to published date.
- Add new 'Fact Check' feed.
- Add support for Twitter, Youtube, Instagram embed.
- Add alt text for each images (not include those chart/graphic).

csisoap committed:
[ReutersBridge] Add new wireitem template type.

- Add new wireitem template type and change the code on line 233 to use the extracted data instead of original one.
- Refactor some codebase.

csisoap committed:
Retain the list of parameters

csisoap committed:
[ReutersBridge] Change timestamp, add new feed, add alt text to image.

- Change timestamp for each article from modified date to published date.
- Add new 'Fact Check' feed.
- Add support for Twitter embed.
- Add alt text for each images (not include those chart/graphic).

csisoap committed:
Revert "[ReutersBridge] Change timestamp, add new feed, add alt text to image."

This reverts commit 5a7b7d8c6aab3907f00e1c4793d24bbb9e9378be.

csisoap committed:
Merge branch 'RSS-Bridge:master' into reuters

csisoap committed:
[ReutersBridge] Change timestamp, add new feed, add alt text to image.

- Change timestamp for each article from modified date to published date.
- Add new 'Fact Check' feed.
- Add support for Twitter embed.
- Add alt text for each images (not include those chart/graphic).

em92 commented on bridges/ReutersBridge.php:
```diff
@@ -202,6 +205,22 @@ private function handleArticleContent($contents) {
+													allowtransparency=\"true\" 
+													allowfullscreen=\"true\" 
+													style=\"width: 550px;height: 225px;\">
+											</iframe>";
```
Why so much indent?

csisoap replied:
I try to indent like this in VS Code
![image](https://user-images.githubusercontent.com/33269526/121107057-6f3aa880-c831-11eb-89f4-56b982c0d92f.png)

but it turn out like that, weird.

csisoap replied:
Indent issue is fixed in new commit.

csisoap committed:
[ReutersBridge] Fix indentation

csisoap committed:
[ReutersBridge] Add `break` statement on line 233

csisoap committed:
[ReutersBridge] Add support for Youtube, Instagram embed.

- Use switch case instead of if else on line 212.
- Add support for Youtube, Instagram embed.

csisoap committed:
Resolve conflict

csisoap committed:
[ReutersBridge] Fix typo

csisoap committed:
[ReutersBridge] Fix indent

em92 commented:
I suggest to use multi line strings instead of those indents. Here is an example
https://github.com/RSS-Bridge/rss-bridge/blob/76c38332ee877986f13bd64fc34b4228ee8fccec/bridges/InternetArchiveBridge.php#L138-L141

csisoap committed:
[ReutersBridge] Rewrite multi-line strings

csisoap committed:
[ReutersBridge] fix CI

csisoap committed:
[ReutersBridge] run PHPCBF

csisoap committed:
Revert "[ReutersBridge] run PHPCBF"

This reverts commit 53f111c69974199beae43786a5e627c413d78d77.

csisoap committed:
[ReutersBridge] Attempt to fix CI

em92 commented:
Just checked those embed iframes. How do you use them? They don't seem to be convenient to use.
![Capture 2](https://user-images.githubusercontent.com/15870400/124977355-d2467780-e049-11eb-8b2c-83e7881d532c.png)

csisoap commented:
> Just checked those embed iframes. How do you use them? They don't seem to be convenient to use.
> ![Capture 2](https://user-images.githubusercontent.com/15870400/124977355-d2467780-e049-11eb-8b2c-83e7881d532c.png)

It's missing the iframe closing tag. That's weird, i remember that add them.

em92 commented:
> It's missing the iframe closing tag. That's weird, i remember that add them.

I don't think problem is about closing tags. Actually iframes are escaped:
https://github.com/RSS-Bridge/rss-bridge/blob/ecaae735d97e240c809cd1fcda06c86675b6d659/lib/FormatAbstract.php#L176-L183

csisoap commented:
> > It's missing the iframe closing tag. That's weird, i remember that add them.
> 
> I don't think problem is about closing tags. Actually iframes are escaped:
> https://github.com/RSS-Bridge/rss-bridge/blob/ecaae735d97e240c809cd1fcda06c86675b6d659/lib/FormatAbstract.php#L176-L183

I think what make iframe not working is this: `<&zwnj;iframe` instead of `<iframe`.
When use Feed Reader like TT-RSS, they cut it out iframe on feeds from RSS-Bridge, but allow iframe from others.

em92 commented:
Ok, I will accept it as is. Good job!
